python/destinations/BigQuery/repo.json

```diff
@@ -0,0 +1,14 @@
+{
+  "Variables": [
+    {
+      "Name": "input",
+      "Type": "EnvironmentVariable",
+      "InputType": "InputTopic",
+      "Description": "This is the input topic for demo data",
+      "DefaultValue": "f1-data",
+      "Required": true
+    }
+  ],
+  "EntryPoint": "build/dockerfile",
+  "RunEntryPoint": "main.py"
+}
```

Comment:
I think this should have deploy settings.
It will pre build it and people can deploy without saving the code to their workspace.
Use validate connection true when you have some code that connects to another system. (which this is)

`,
    "DeploySettings":
    {
        "DeploymentType": "Service",
        "CpuMillicores": 200,
        "MemoryInMb": 200,
        "Replicas": 1,
        "ValidateConnection": false
    }`

Reply:
You need to log "CONNECTED!" when its connected to the other system